packages/ubuntu_desktop_installer/test/keyboard_layout/keyboard_layout_page_test.dart

```diff
@@ -85,12 +85,14 @@ void main() {
     await tester.pumpWidget(tester.buildApp((_) => buildPage(model)));
 
     for (var i = 0; i < 3; ++i) {
-      final variantTile = find.widgetWithText(ListTile, 'Variant $i');
-      expect(variantTile, findsOneWidget);
-      await tester.tap(variantTile);
+      await tester.tap(find.byType(MenuButtonBuilder<int>));
+      await tester.pumpAndSettle();
+      final variantItem = find.widgetWithText(MenuItemButton, 'Variant $i');
+      expect(variantItem, findsOneWidget);
+      await tester.tap(variantItem);
       verify(model.selectVariant(i)).called(1);
     }
-  });
+  }, skip: true); // TODO: fix the test
```

Comment:
i'm having trouble figuring out why `tap(MenuButtonBuilder)` doesn't work :(

```

Warning: A call to tap() with finder "exactly one widget with type "MenuButtonBuilder<int>" (ignoring offstage widgets): MenuButtonBuilder<int>(dependencies: [Directionality, MediaQuery, _InheritedTheme, _LocalizationsScope-[GlobalKey#79b53]], state: _MenuButtonBuilderState<int>#71692)" derived an Offset (Offset(609.0, 451.0)) that would not hit test on the specified widget.
Maybe the widget is actually off-screen, or another widget is obscuring it, or the widget cannot receive pointer events.
```

Reply:
it works when I only test the very first variant without a loop - I suspect the issue is that the test tries to tap the `MenuButtonBuilder<int>` again, while the popup is visible

Reply:
Moving
```dart
await tester.tap(find.byType(MenuButtonBuilder<int>));
await tester.pumpAndSettle();
```
out of the loop works

Reply:
ahh, i'm an idiot :D thank you!